apps/wing/src/cli.ts

```diff
@@ -36,6 +36,8 @@ async function main() {
     process.env.PROGRESS = "1";
   });
 
+  program.option("--no-progress", "Don't show compilation progress", () => {});
```

Comment:
Not sure this is needed

```suggestion
  program.option("--no-progress", "Don't show compilation progress");
```